extensions/azure/cosmos/assetindex-cosmos/src/test/java/org/eclipse/dataspaceconnector/assetindex/azure/CosmosAssetIndexTest.java

```diff
@@ -248,6 +249,24 @@ void findAll_withFiltering() {
         verify(api).queryItems(any(SqlQuerySpec.class));
     }
 
+    @Test
+    void deleteById_whenPresent_deletesItem() {
+        String id = "id-test";
+        AssetDocument document = createDocument(id);
+        when(api.deleteItem(eq(id))).thenReturn(document);
```

Comment:
```suggestion
        when(api.deleteItem(id)).thenReturn(document);
```
nit: no need for eq() in simple cases